backend/assessments/1/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,77 @@
+package org.example;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Main {
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
+    public static void main(String[] args) {
+        String csvPath = "src/main/resources/IPL_2021-data.csv";
```

Comment:
Hardcoded paths

Reply:
Sagun tomorrow please explain this path waala part to me, I made the same mistake in the assignment also...